### PR TITLE
[US-38] Front-end: Add development cards button and panel

### PR DESCRIPTION
**Pull request by andrii-r**

### CODE REVIEW

File: src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+            return {
+                restrict: 'E',
+                scope: {
+                    game: "="
+                },
+                templateUrl: "/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html",
+                link: function(scope) {
+                    scope.cards = function() {
+                        var currentGameUser = scope.game.getCurrentUser();
+                        return (currentGameUser) ? currentGameUser.developmentCards : {};
```
Comment (andrii-r) [inline src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.js:15]: why do we need this check? should be there any case when currentGameUser is undefined?

File: src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<ct-modal-window modal-window-id="CARDS_PANEL">
+
+    <div class="ct-modal-window-bg" ct-modal-window-close></div>
+    <div class="ct-modal-window-content content">
+
+        <button ng-disabled="cards().knight===0">
```
Comment (andrii-r) [inline src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html:6]: we should disable buttons, but show quantity value when it is not user's turn - now it is possible to click on the button if it is not user's turn

Comment (skisor) [inline src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html:6]: The buttons are dummies. When using of dev cards appears, new available actions will appear too, and the buttons will be enabled/disabled depend on them. For now, "disabling" using only for showing buttons' style

File: src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.js (hunk 1 above)
Comment (skisor) [inline src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.js:15]: When a spectators are added to the game. If functionality of spectators is needed in the future

Comment (andrii-r) [inline src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.js:15]: they will not have 'cards' button available at all, so why do we need this empty list?

File: src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html (hunk 2 above)
Comment (andrii-r) [inline src/main/webapp/features/screens/game/dashboard/cardsPanel/ct-cards-panel.html:6]: you already have enabled/disabled functionality, but if you are going to implement third step: 'clickable', in such case leave it as is

**Pull request merged**